### DISCUSSION TITLE
Howto Update virtual host folder in Webview2?

### DISCUSSION THREAD

**Discussion by gnobbi** [MicrosoftEdge/WebView2Feedback#3933] [Q&A]
am using wpf with the New webview2 panel.

I want to show local PDFs in the webview2. I Set up an virtual host "./www/test.pdf" - > https://locale.example/test.pdf folder it works fine.

During the process I overwrite. ./www/test.pdf but it Do Not refresh in the view. Also reload and clear and Re add the virtual Host folder Do Not work.

Has someone an idea?

Or an other approach?

Thank you in advance

### Edit:

I set up the virtual host with ./www not ./www/test.pdf

This is also posted in stackoverflow.com
 https://stackoverflow.com/q/70882598/12273211

**Comment by champnic** [comment]
Right now it's a limitation of the API that it requires a refresh before it takes effect. It should apply to subsequent navigations. Are you seeing a different behavior?
